dotnet/CoreLib/DataFormats/Image/AzureFormRecognizer/AzureFormRecognizerConfig.cs

```diff
@@ -0,0 +1,10 @@
+﻿// Copyright (c) Microsoft. All rights reserved.
+
+namespace Microsoft.SemanticMemory.DataFormats.Image.AzureFormRecognizer;
+
+public class AzureFormRecognizerConfig
```

Comment:
could you add AAD like the other Azure services? e.g. see `AzureTextGeneration` and `AzureOpenAIConfig`